lib/grammars.coffee

```diff
@@ -497,6 +497,11 @@ module.exports =
     "File Based":
       command: "powershell"
       args: (context) -> [context.filepath.replace /\ /g, "` "]
+      
+  Processing:
+    "File Based":
+      command: "bash"
+      args: (context) -> ['-c', 'processing-java --sketch='+context.filepath.replace("/"+context.filename,"")+' --run']
```

Comment:
I don't think it will work on windows atm, I have access to windows today so I will update the code today.